core/src/test/java/org/elasticsearch/common/ParseFieldTests.java

```diff
@@ -18,45 +18,67 @@
  */
 package org.elasticsearch.common;
 
+import org.elasticsearch.common.ParseField.DeprecationHandler;
 import org.elasticsearch.test.ESTestCase;
+import org.mockito.ArgumentCaptor;
 
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.not;
 import static org.hamcrest.CoreMatchers.sameInstance;
 import static org.hamcrest.collection.IsArrayContainingInAnyOrder.arrayContainingInAnyOrder;
+import static org.mockito.Mockito.eq;
```

Comment:
Leftovers?

Reply:
Yeah. You need to use `eq` if you are capturing arguments. You can't just send in an object.

Reply:
And I'll remove it.